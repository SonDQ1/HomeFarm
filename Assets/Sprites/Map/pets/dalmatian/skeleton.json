{"bones":[{"name":"root"},{"name":"bone","parent":"root","length":97.59,"x":-32.18,"y":3.27,"rotation":90},{"name":"body","parent":"bone","length":120.42,"x":142.28,"y":-166.44,"rotation":110.85},{"name":"thigh_back","parent":"bone","length":52.34,"x":140.95,"y":-156.37,"rotation":-156.57},{"name":"thigh_back2","parent":"bone","length":39.39,"x":149.75,"y":-134.23,"rotation":-176.23},{"name":"thigh_front","parent":"bone","length":50.66,"x":94.6,"y":-76.43,"rotation":-177.74},{"name":"thigh_front2","parent":"bone","length":41.95,"x":106.1,"y":-45.09,"rotation":-179.96},{"name":"head","parent":"body","length":47.29,"x":110.73,"y":-23.39,"rotation":-58.36},{"name":"leg_back","parent":"thigh_back","length":59.18,"x":45.28,"y":0.01,"rotation":-19.99},{"name":"leg_back2","parent":"thigh_back2","length":60.39,"x":39.74,"y":-1.36,"rotation":-0.32},{"name":"leg_front","parent":"thigh_front","length":46.66,"x":47.49,"y":1.78,"rotation":-5.39},{"name":"leg_front2","parent":"thigh_front2","length":44.96,"x":40.9,"y":0.38,"rotation":-0.92},{"name":"tail","parent":"body","length":64.85,"x":-14.04,"y":-18.68,"rotation":116.19},{"name":"ear","parent":"head","length":21.32,"x":36.29,"y":-36.47,"rotation":133.86},{"name":"ear2","parent":"head","length":23.74,"x":68.43,"y":-25.78,"rotation":157.35}],"slots":[{"name":"thigh_back2","bone":"thigh_back2","attachment":"thigh_back2"},{"name":"leg_back2","bone":"leg_back2","attachment":"leg_back2"},{"name":"leg_front2","bone":"leg_front2","attachment":"leg_front2"},{"name":"thigh_front2","bone":"thigh_front2","attachment":"thigh_front2"},{"name":"tail","bone":"tail","attachment":"tail"},{"name":"body","bone":"body","attachment":"body"},{"name":"ear2","bone":"ear2","attachment":"ear2"},{"name":"head","bone":"head","attachment":"head"},{"name":"ear","bone":"ear","attachment":"ear"},{"name":"leg_front","bone":"leg_front","attachment":"leg_front"},{"name":"thigh_front","bone":"thigh_front","attachment":"thigh_front"},{"name":"thigh_back","bone":"thigh_back","attachment":"thigh_back"},{"name":"leg_back","bone":"leg_back","attachment":"leg_back"}],"skins":{"default":{"body":{"body":{"x":60.62,"y":-0.73,"scaleX":1.054,"scaleY":0.986,"rotation":159.14,"width":149,"height":112}},"ear":{"ear":{"x":10.76,"y":-2.53,"rotation":83.64,"width":25,"height":35}},"ear2":{"ear2":{"x":12.66,"y":-2.29,"rotation":60.15,"width":29,"height":38}},"head":{"head":{"x":32.85,"y":-4.61,"scaleX":0.987,"scaleY":0.987,"rotation":-142.48,"width":82,"height":90}},"leg_back":{"leg_back":{"x":27.94,"y":1.13,"rotation":80.25,"width":36,"height":78}},"leg_back2":{"leg_back2":{"x":28.49,"y":-1.31,"scaleX":0.942,"scaleY":0.942,"rotation":77.49,"width":36,"height":78}},"leg_front":{"leg_front":{"x":21.75,"y":-3.91,"rotation":93.14,"width":26,"height":63}},"leg_front2":{"leg_front2":{"x":22.26,"y":-3.22,"rotation":90.89,"width":26,"height":63}},"tail":{"tail":{"x":37.56,"y":1.24,"rotation":-33.02,"width":63,"height":43}},"thigh_back":{"thigh_back":{"x":16.06,"y":-0.29,"rotation":58.4,"width":57,"height":80}},"thigh_back2":{"thigh_back2":{"x":13.75,"y":-5.26,"scaleX":0.942,"scaleY":0.942,"rotation":64.85,"width":57,"height":80}},"thigh_front":{"thigh_front":{"x":21.93,"y":1.37,"rotation":87.74,"width":32,"height":66}},"thigh_front2":{"thigh_front2":{"x":16.51,"y":1.24,"rotation":89.96,"width":32,"height":66}}}},"animations":{"animation":{"bones":{"root":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.8333,"x":0,"y":0,"curve":"stepped"},{"time":1.5,"x":0,"y":0,"curve":"stepped"},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"bone":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.8333,"x":0,"y":0,"curve":"stepped"},{"time":1.5,"x":0,"y":0,"curve":"stepped"},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"body":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0},{"time":1.5,"angle":-1.98},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.8333,"x":-2.9,"y":0,"curve":"stepped"},{"time":1.5,"x":-2.9,"y":0},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"thigh_back":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.8333,"x":-1.16,"y":0,"curve":"stepped"},{"time":1.5,"x":-1.16,"y":0},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"thigh_back2":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.8333,"x":-3.48,"y":0,"curve":"stepped"},{"time":1.5,"x":-3.48,"y":0},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"thigh_front":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.8333,"x":-2.9,"y":0},{"time":1.5,"x":0.58,"y":0},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"thigh_front2":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.8333,"x":-2.9,"y":0},{"time":1.5,"x":0.58,"y":0},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"head":{"rotate":[{"time":0,"angle":0},{"time":0.8333,"angle":2.94},{"time":1.5,"angle":-6.54},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.8333,"x":0,"y":0,"curve":"stepped"},{"time":1.5,"x":0,"y":0,"curve":"stepped"},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"leg_back":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.8333,"x":-1.06,"y":0.46,"curve":"stepped"},{"time":1.5,"x":-1.06,"y":0.46},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"leg_back2":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.8333,"x":-3.47,"y":0.22,"curve":"stepped"},{"time":1.5,"x":-3.47,"y":0.22},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"leg_front":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.8333,"x":-2.9,"y":0.11},{"time":1.5,"x":0.58,"y":-0.02},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"leg_front2":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0,"curve":"stepped"},{"time":1.5,"angle":0,"curve":"stepped"},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.8333,"x":-2.9,"y":0},{"time":1.5,"x":0.58,"y":0},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"tail":{"rotate":[{"time":0,"angle":0},{"time":0.8333,"angle":7.43},{"time":1.5,"angle":-9.36},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.8333,"x":0,"y":0,"curve":"stepped"},{"time":1.5,"x":0,"y":0,"curve":"stepped"},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"ear":{"rotate":[{"time":0,"angle":0},{"time":0.8333,"angle":-0.11},{"time":1.5,"angle":20.35},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.8333,"x":0,"y":0},{"time":1.5,"x":-0.4,"y":-0.41},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]},"ear2":{"rotate":[{"time":0,"angle":0},{"time":0.8333,"angle":-17.85},{"time":1.5,"angle":7.52},{"time":2.3333,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.8333,"x":0,"y":0,"curve":"stepped"},{"time":1.5,"x":0,"y":0,"curve":"stepped"},{"time":2.3333,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.8333,"x":1,"y":1,"curve":"stepped"},{"time":1.5,"x":1,"y":1,"curve":"stepped"},{"time":2.3333,"x":1,"y":1}]}}}}}