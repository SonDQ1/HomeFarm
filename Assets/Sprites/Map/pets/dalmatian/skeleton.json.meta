fileFormatVersion: 2
guid: d93aff2a691f01547ab121f30302490b
timeCreated: 1550282843
licenseType: Pro
TextScriptImporter:
  userData: 
  assetBundleName: 
  assetBundleVariant: 
