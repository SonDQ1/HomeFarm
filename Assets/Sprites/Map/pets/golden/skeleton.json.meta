fileFormatVersion: 2
guid: 8c27dc5c8fb60834598bee144ac9ae8c
timeCreated: 1550282843
licenseType: Pro
TextScriptImporter:
  userData: 
  assetBundleName: 
  assetBundleVariant: 
