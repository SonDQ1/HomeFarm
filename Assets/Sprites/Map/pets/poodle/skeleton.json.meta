fileFormatVersion: 2
guid: 85993c53f9fc496468f8c3aaead2af12
timeCreated: 1550282843
licenseType: Pro
TextScriptImporter:
  userData: 
  assetBundleName: 
  assetBundleVariant: 
