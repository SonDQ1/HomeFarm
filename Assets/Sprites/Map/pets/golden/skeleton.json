{"bones":[{"name":"root"},{"name":"bone","parent":"root","length":99.94,"x":-31.63,"y":1.1,"rotation":90.45},{"name":"body","parent":"bone","length":137.18,"x":164.75,"y":-193.77,"rotation":111.09},{"name":"thigh_back","parent":"bone","length":53.8,"x":164.83,"y":-183.43,"rotation":-156.2},{"name":"thigh_back2","parent":"bone","length":45,"x":162.97,"y":-142.62,"rotation":-153.97},{"name":"thigh_front","parent":"bone","length":55.37,"x":118.21,"y":-93.3,"rotation":-169.73},{"name":"thigh_front2","parent":"bone","length":51.45,"x":135.35,"y":-58.62,"rotation":-172.8},{"name":"head","parent":"body","length":86.17,"x":118.85,"y":-13,"rotation":-87.27},{"name":"leg_back","parent":"thigh_back","length":49.32,"x":60.6,"y":0.74,"rotation":-36.27},{"name":"leg_back2","parent":"thigh_back2","length":48.56,"x":46.47,"y":1.77,"rotation":-37.08},{"name":"leg_front","parent":"thigh_front","length":56.84,"x":57.41,"y":-0.89,"rotation":-15.82},{"name":"leg_front2","parent":"thigh_front2","length":61.96,"x":50.61,"y":0.87,"rotation":-8.67},{"name":"tail","parent":"body","length":62.04,"x":-5.43,"y":-27.09,"rotation":99.92},{"name":"ear","parent":"head","length":40.43,"x":68.99,"y":-20.36,"rotation":161.15},{"name":"ear2","parent":"head","length":45.78,"x":100.83,"y":5.26,"rotation":162.13}],"slots":[{"name":"ear2","bone":"ear2","attachment":"ear2"},{"name":"leg_front2","bone":"leg_front2","attachment":"leg_front2"},{"name":"thigh_front2","bone":"thigh_front2","attachment":"thigh_front2"},{"name":"leg_back2","bone":"leg_back2","attachment":"leg_back2"},{"name":"thigh_back2","bone":"thigh_back2","attachment":"thigh_back2"},{"name":"tail","bone":"tail","attachment":"tail"},{"name":"body","bone":"body","attachment":"body"},{"name":"leg_front","bone":"leg_front","attachment":"leg_front"},{"name":"thigh_front","bone":"thigh_front","attachment":"thigh_front"},{"name":"head","bone":"head","attachment":"head"},{"name":"ear","bone":"ear","attachment":"ear"},{"name":"leg_back","bone":"leg_back","attachment":"leg_back"},{"name":"thigh_back","bone":"thigh_back","attachment":"thigh_back"}],"skins":{"default":{"body":{"body":{"x":75.47,"y":-1.64,"rotation":153.62,"width":174,"height":140}},"ear":{"ear":{"x":19.26,"y":-1.42,"rotation":82.27,"width":34,"height":56}},"ear2":{"ear2":{"x":19.83,"y":-0.01,"rotation":81.29,"width":36,"height":56}},"head":{"head":{"x":28.01,"y":12.2,"rotation":-116.57,"width":111,"height":147}},"leg_back":{"leg_back":{"x":22.45,"y":0.02,"rotation":102.02,"width":32,"height":59}},"leg_back2":{"leg_back2":{"x":22.86,"y":0.52,"rotation":100.61,"width":32,"height":59}},"leg_front":{"leg_front":{"x":18.39,"y":0.32,"rotation":95.1,"width":30,"height":89}},"leg_front2":{"leg_front2":{"x":27.75,"y":-0.63,"scaleX":0.977,"scaleY":0.977,"rotation":91.02,"width":30,"height":89}},"tail":{"tail":{"x":35.17,"y":4.25,"rotation":38.55,"width":76,"height":51}},"thigh_back":{"thigh_back":{"x":20.22,"y":-4.43,"rotation":65.75,"width":60,"height":82}},"thigh_back2":{"thigh_back2":{"x":16.02,"y":-3.14,"rotation":63.52,"width":60,"height":82}},"thigh_front":{"thigh_front":{"x":20.07,"y":1.56,"rotation":79.28,"width":52,"height":89}},"thigh_front2":{"thigh_front2":{"x":20.98,"y":1.17,"scaleX":0.977,"scaleY":0.977,"rotation":82.34,"width":52,"height":89}}}},"animations":{"stand":{"bones":{"root":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6666,"angle":0,"curve":"stepped"},{"time":1.1666,"angle":0,"curve":"stepped"},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.6666,"x":0,"y":0,"curve":"stepped"},{"time":1.1666,"x":0,"y":0,"curve":"stepped"},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"bone":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6666,"angle":0,"curve":"stepped"},{"time":1.1666,"angle":0,"curve":"stepped"},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.6666,"x":0,"y":0,"curve":"stepped"},{"time":1.1666,"x":0,"y":0,"curve":"stepped"},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"body":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6666,"angle":0},{"time":1.1666,"angle":-3.27},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.6666,"x":-2.17,"y":0.01,"curve":"stepped"},{"time":1.1666,"x":-2.17,"y":0.01},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"thigh_back":{"rotate":[{"time":0,"angle":0},{"time":0.6666,"angle":4.84,"curve":"stepped"},{"time":1.1666,"angle":4.84},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.6666,"x":-4.87,"y":2.21,"curve":"stepped"},{"time":1.1666,"x":-4.87,"y":2.21},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"thigh_back2":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6666,"angle":0,"curve":"stepped"},{"time":1.1666,"angle":0,"curve":"stepped"},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.6666,"x":-3.24,"y":2.74,"curve":"stepped"},{"time":1.1666,"x":-3.24,"y":2.74},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"thigh_front":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6666,"angle":0,"curve":"stepped"},{"time":1.1666,"angle":0,"curve":"stepped"},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.6666,"x":-3.8,"y":0.02},{"time":1.1666,"x":0,"y":1.08},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"thigh_front2":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6666,"angle":0,"curve":"stepped"},{"time":1.1666,"angle":0,"curve":"stepped"},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.6666,"x":-3.28,"y":-2.15},{"time":1.1666,"x":-0.01,"y":-1.63},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"head":{"rotate":[{"time":0,"angle":0},{"time":0.6666,"angle":3.15},{"time":1.1666,"angle":0.74},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.6666,"x":0,"y":0},{"time":1.1666,"x":0.39,"y":4.29},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"leg_back":{"rotate":[{"time":0,"angle":0},{"time":0.6666,"angle":-4.84,"curve":"stepped"},{"time":1.1666,"angle":-4.84},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.6666,"x":-3.37,"y":-0.85,"curve":"stepped"},{"time":1.1666,"x":-3.37,"y":-0.85},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"leg_back2":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6666,"angle":0,"curve":"stepped"},{"time":1.1666,"angle":0,"curve":"stepped"},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.6666,"x":-1.7,"y":3.89,"curve":"stepped"},{"time":1.1666,"x":-1.7,"y":3.89},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"leg_front":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6666,"angle":0,"curve":"stepped"},{"time":1.1666,"angle":0,"curve":"stepped"},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.6666,"x":-3.74,"y":0.7},{"time":1.1666,"x":0.2,"y":1.06},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"leg_front2":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6666,"angle":0,"curve":"stepped"},{"time":1.1666,"angle":0,"curve":"stepped"},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.6666,"x":-3.52,"y":-1.72},{"time":1.1666,"x":-0.21,"y":-1.61},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"tail":{"rotate":[{"time":0,"angle":0},{"time":0.6666,"angle":-13.17},{"time":1.1666,"angle":14.98},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.6666,"x":0,"y":0},{"time":1.1666,"x":0.98,"y":-0.17},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"ear":{"rotate":[{"time":0,"angle":0},{"time":0.6666,"angle":-15.07},{"time":1.1666,"angle":-7.21},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0,"curve":"stepped"},{"time":0.6666,"x":0,"y":0,"curve":"stepped"},{"time":1.1666,"x":0,"y":0,"curve":"stepped"},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]},"ear2":{"rotate":[{"time":0,"angle":0},{"time":0.6666,"angle":3.76},{"time":1.1666,"angle":11},{"time":2,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.1666,"x":-0.34,"y":-0.74},{"time":0.6666,"x":0,"y":0},{"time":1.1666,"x":-2.26,"y":0.3},{"time":2,"x":0,"y":0}],"scale":[{"time":0,"x":1,"y":1,"curve":"stepped"},{"time":0.6666,"x":1,"y":1,"curve":"stepped"},{"time":1.1666,"x":1,"y":1,"curve":"stepped"},{"time":2,"x":1,"y":1}]}}}}}